### PR TITLE
Added Troubleshooting section

### PR DESCRIPTION
**Pull request by slyubimt**
### Details:
 - *Added Troubleshooting section*

### Tickets:
 - *CVS-55393*
 - *CVS-54498*

### CODE REVIEW

File: docs/install_guides/pypi-openvino-dev.md [hunk 1]
```diff
@@ -73,22 +69,22 @@ On Windows:
 openvino_env\Scripts\activate
 ```
 
-### Step 4. Set Up and Update pip to the Highest Version
+### Step 3. Set Up and Update pip to the Highest Version
```
Comment (tsavina) [inline docs/install_guides/pypi-openvino-dev.md:72]: Step 3. Set Up and Update pip to the Highest Version ---> Step 3. Set Up and Update Pip to the Highest Version
all words in headings except articles and prepositions are usually capitalized

File: docs/install_guides/pypi-openvino-dev.md [hunk 2]
```diff
@@ -97,6 +93,19 @@ pot -h
...
 
+## Troubleshooting
+
+#### Error: Microsoft Visual C++ 14.0 is required. Get it with "Build Tools for Visual Studio"
+
+On Windows* some dependencies may require compilation from source when installing. To resolve this issue you are required to install [Build Tools for Visual Studio* 2019](https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2019) and repete package installation.
+
+#### ImportError: libpython3.7m.so.1.0: cannot open shared object file: No such file or directory
+
+To resolve missing external dependency on Ubuntu* execute a command:
```
Comment (tsavina) [inline docs/install_guides/pypi-openvino-dev.md:104]: To resolve missing external dependency on Ubuntu* execute a command: ---> To resolve missing external dependency on Ubuntu*, execute the following command:

File: docs/install_guides/pypi-openvino-dev.md [hunk 3]
```diff
@@ -97,6 +93,19 @@ pot -h
 
 You will see the help message for Post-Training Optimization Tool if installation finished successfully.
 
+## Troubleshooting
+
+#### Error: Microsoft Visual C++ 14.0 is required. Get it with "Build Tools for Visual Studio"
+
+On Windows* some dependencies may require compilation from source when installing. To resolve this issue you are required to install [Build Tools for Visual Studio* 2019](https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2019) and repete package installation.
```
Comment (tsavina) [inline docs/install_guides/pypi-openvino-dev.md:100]: To resolve this issue you are required to install ... and repete package installation ---> To resolve this issue, you need to install .... and repeat package installation

File: docs/install_guides/pypi-openvino-rt.md [hunk 4]
```diff
@@ -66,22 +62,22 @@ On Windows:
 openvino_env\Scripts\activate
 ```
 
-### Step 4. Set Up and Update pip to the Highest Version
+### Step 3. Set Up and Update pip to the Highest Version
```
Comment (tsavina) [inline docs/install_guides/pypi-openvino-rt.md:65]: Pip

**Review by ryanloney — APPROVED** [review]

File: docs/install_guides/pypi-openvino-rt.md [hunk 5]
```diff
@@ -90,6 +86,19 @@ python -c "from openvino.inference_engine import IECore"
...
 
+## Troubleshooting
+
+#### Error: Microsoft Visual C++ 14.0 is required. Get it with "Build Tools for Visual Studio"
+
+On Windows* some dependencies may require compilation from source when installing. To resolve this issue you are required to install [Build Tools for Visual Studio* 2019](https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2019) and repete package installation.
+
+#### ImportError: libpython3.7m.so.1.0: cannot open shared object file: No such file or directory
+
+To resolve missing external dependency on Ubuntu* execute a command:
```
Comment (tsavina) [inline docs/install_guides/pypi-openvino-rt.md:97]: To resolve missing external dependency on Ubuntu* execute a command: ---> To resolve missing external dependency on Ubuntu*, execute the following command:

File: docs/install_guides/pypi-openvino-rt.md [hunk 6]
```diff
@@ -90,6 +86,19 @@ python -c "from openvino.inference_engine import IECore"
    
 You will not see any error messages if installation finished successfully.
 
+## Troubleshooting
+
+#### Error: Microsoft Visual C++ 14.0 is required. Get it with "Build Tools for Visual Studio"
+
+On Windows* some dependencies may require compilation from source when installing. To resolve this issue you are required to install [Build Tools for Visual Studio* 2019](https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2019) and repete package installation.
```
Comment (tsavina) [inline docs/install_guides/pypi-openvino-rt.md:93]: To resolve this issue you are required to install ... and repete package installation ---> To resolve this issue, you need to install .... and repeat package installation

**Comment by ryanloney** [comment]
@helena-intel can you please review before it is merged. I did not check all steps, just approving the move the C++ to troubleshooting

File: docs/install_guides/pypi-openvino-dev.md (hunk 1 above)
Comment (slyubimt) [reply docs/install_guides/pypi-openvino-dev.md]: pip is a recursive acronym for "Pip Installs Packages", so I believe all letters shall be capitalized.

**Review by tsavina — APPROVED** [review]

**Pull request merged**